### PR TITLE
adding organizer

### PR DESCRIPTION
**Pull request by adityadhoot**
Adding photo and bio

### CODE REVIEW

File: _organizers/aditya-dhoot.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+github: adityadhoot
+layout: base
+name: Aditya Dhoot
+photo_url: /static/img/organizers/adityad.jpg
```
Comment (tim-schilling) [inline _organizers/aditya-dhoot.md:5]: I'm reviewing from my phone so I can't run this, but it's safer to make sure the casing matches on the file.

File: _organizers/aditya-dhoot.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+role: Conference Reviewer
```
Comment (tim-schilling) [inline _organizers/aditya-dhoot.md:6]: I think the typical term is Program team.

File: _organizers/aditya-dhoot.md (hunk 1 above)
Comment (adityadhoot) [reply _organizers/aditya-dhoot.md]: fixed

File: _organizers/aditya-dhoot.md (hunk 2 above)
Comment (adityadhoot) [reply _organizers/aditya-dhoot.md]: fixed

**Review by tim-schilling — APPROVED** [review]
Thank you!

**Pull request merged**